ledfx/presets.py

```diff
@@ -2087,4 +2087,60 @@
             "name": "Reset",
         }
     },
+    "scan": {
+        "reset": {
+            "config": {
+                "advanced": False,
+                "background_brightness": 1.0,
+                "background_color": "#000000",
+                "blur": 0.0,
+                "bounce": True,
+                "brightness": 1.0,
+                "color_intensity": False,
+                "color_scan": "#ff0000",
+                "count": 1,
+                "flip": False,
+                "frequency_range": "Lows (beat+bass)",
+                "full_grad": False,
+                "gradient": "linear-gradient(90deg, rgb(255, 0, 0) 0%, rgb(255, 120, 0) 14%, rgb(255, 200, 0) 28%, rgb(0, 255, 0) 42%, rgb(0, 199, 140) 56%, rgb(0, 0, 255) 70%, rgb(128, 0, 128) 84%, rgb(255, 0, 178) 98%)",
+                "gradient_roll": 0.0,
+                "mirror": False,
+                "modulate": False,
+                "modulation_effect": "sine",
+                "modulation_speed": 0.5,
+                "multiplier": 3.0,
+                "scan_width": 30,
+                "speed": 50,
+                "use_grad": False,
+            },
+            "name": "Reset",
+        },
+        "painbow": {
+            "config": {
+                "advanced": True,
+                "background_brightness": 0.0,
+                "background_color": "#000000",
+                "blur": 0.0,
+                "bounce": False,
+                "brightness": 1.0,
+                "color_intensity": True,
+                "color_scan": "#ff0000",
+                "count": 2,
+                "flip": False,
+                "frequency_range": "Lows (beat+bass)",
+                "full_grad": True,
+                "gradient": "linear-gradient(90deg, rgb(255, 0, 0) 0%, rgb(255, 120, 0) 14%, rgb(255, 200, 0) 28%, rgb(0, 255, 0) 42%, rgb(0, 199, 140) 56%, rgb(0, 0, 255) 70%, rgb(128, 0, 128) 84%, rgb(255, 0, 178) 98%)",
+                "gradient_roll": 0.5,
+                "mirror": False,
+                "modulate": False,
+                "modulation_effect": "sine",
+                "modulation_speed": 0.65,
+                "multiplier": 3.0,
+                "scan_width": 30,
+                "speed": 31,
+                "use_grad": True,
+            },
+            "name": "painbow",
```

Comment:
Rainbow?

Reply:
Intentionally painbow, but maybe I am being dry. Humor mode

Meanwhile I note I have double defined scan, so I need to unwind thay